src/Controls/src/Core/ReferenceTypeConverter.cs

```diff
@@ -10,6 +10,7 @@
 namespace Microsoft.Maui.Controls
 {
 	/// <include file="../../docs/Microsoft.Maui.Controls/ReferenceTypeConverter.xml" path="Type[@FullName='Microsoft.Maui.Controls.ReferenceTypeConverter']/Docs/*" />
+	[RequireService([typeof(IReferenceProvider), typeof(IProvideParentValues)])]
```

Comment:
this is the only IExtendedTypeConverter that needs to be attributed, all the others have a compiled counterpart